pkg/state/state.go

```diff
@@ -3317,13 +3324,14 @@ func (ar *AffectedReleases) DisplayAffectedReleases(logger *zap.SugaredLogger) {
 	if ar.Failed != nil && len(ar.Failed) > 0 {
 		logger.Info("\nFAILED RELEASES:")
 		tbl, _ := prettytable.NewTable(prettytable.Column{Header: "NAME"},
+			prettytable.Column{Header: "NAMESPACE", MinWidth: 6},
 			prettytable.Column{Header: "CHART", MinWidth: 6},
 			prettytable.Column{Header: "VERSION", MinWidth: 6},
 			prettytable.Column{Header: "DURATION", AlignRight: true},
 		)
 		tbl.Separator = "   "
 		for _, release := range ar.Failed {
-			err := tbl.AddRow(release.Name, release.Chart, release.installedVersion, release.duration.Round(time.Second))
+			err := tbl.AddRow(release.Name, release.NamespaceOrDefault(), release.Chart, release.installedVersion, release.duration.Round(time.Second))
```

Comment:
```suggestion
			err := tbl.AddRow(release.Name, release.Namespace, release.Chart, release.installedVersion, release.duration.Round(time.Second))
```

Any reason we want `NamespaceOrDefault()` instead of `Namespace`?

Reply:
Now Namespace is ok. I will update the code.

Reply:
@mumoshu done